pages/connect-wallets.tsx

```diff
@@ -0,0 +1,70 @@
+import type { NextPage } from 'next';
+import WalletConnector from '../components/WalletConnector';
+import { useContext } from 'react';
+import { useRouter } from 'next/router';
+import { WalletAddressContext } from '../contexts/WalletAddressContext';
+import { toast } from 'react-toastify';
+import { L16_CHAIN_ID } from '../constants';
+
+const ConnectWallets: NextPage = () => {
+  const router = useRouter();
+  const { setWalletAddress } = useContext(WalletAddressContext);
+
+  const verifyChainId = () => {
+    const chainId = parseInt(window.ethereum.networkVersion);
+    if (chainId && window.ethereum.networkVersion != L16_CHAIN_ID) {
+      toast('Please switch to L16 network');
+      return false;
+    } else {
+      return true;
+    }
```

Comment:
```suggestion
    if (chainId && window.ethereum.networkVersion != L16_CHAIN_ID) {
      toast('Please switch to L16 network');
      return false;
    }
    
    return true;
```

you return in the if so no need to add a else. The linter / or TS should tell you